packages/lib/gpc/test/common.ts

```diff
@@ -44,6 +44,7 @@ export const sampleEntries = {
   H: { type: "int", value: 8n },
   I: { type: "int", value: 9n },
   J: { type: "int", value: 10n },
+  K: { type: "int", value: -5n },
```

Comment:
I suggest adding a negative entry to sampleEntries used in other packages/layers as well, since the intent is for it to exercise a variety of types and features.
It might also be easier/better to change one of the existing values rather than add a new one.  It's not relevant at this layer, but at the POD layer I think I picked the number of sample entries to exercise different edge cases of the Merkle tree.